src/core.js

```diff
@@ -11,6 +11,11 @@ nv.charts = {}; //stores all the ready to use charts
 nv.logs = {}; //stores some statistics and potential error messages
 nv.dom = {}; //DOM manipulation functions
 
+// Node/CommonJS - require D3
+if (typeof(module) !== 'undefined' && typeof(exports) !== 'undefined' && typeof(d3) == 'undefined') {
+    d3 = require('d3');
+}
+
```

Comment:
You're talking about the order in which it is loaded? Or whether it's loaded at all? In which case do you think there is going to be a problem?